sections/05_treat.md

```diff
@@ -257,11 +257,11 @@ study of BACE-1 inhibitors included only 1547 compounds
 this limited dataset, but overfitting was a challenge and the differences
 between random forests and a deep neural were negligible, especially in the
```

Comment:
You introduced the RF abbreviation before in this section (more generally, we need to agree whether to use the abbreviation or not and check when to first introduce it).